### ISSUE TITLE
Tiled isn't showing the window list in the dock icon

### ISSUE THREAD

**Issue by vadi2** [bjorn/tiled#1470]
Tiled isn't showing the window list in the dock icon, so when you minimise a window with the [minimise to dock icon enabled](https://www.howtogeek.com/285750/how-to-set-macos-applications-to-minimize-to-their-dock-icon/), there is no way to bring them back save for using "Show all".

**Comment by vadi2** [comment]
[Screencast 2017-03-06 20:18:41.mp4.zip](https://github.com/bjorn/tiled/files/822225/Screencast.2017-03-06.20.18.41.mp4.zip) demo of the issue.

It's a Qt5-specific issue that doesn't happen in Qt4. I'm another Qt dev looking for a solution for this issue and noticed Tiled suffers from the same issue.

**Comment by bjorn** [comment]
> Tiled isn't showing the window list in the dock icon, so when you minimise a window with the minimise to dock icon enabled, there is no way to bring them back save for using "Show all".

Hmm, I've just tried it and I can bring back the window by simply clicking the icon. That should at least work 99% of the time, though a problem still exists with secondary windows like the Tile Collision Editor and the Tile Animation Editor.

> It's a Qt5-specific issue that doesn't happen in Qt4. I'm another Qt dev looking for a solution for this issue and noticed Tiled suffers from the same issue.

Is there a Qt bug report already open about this?

**Comment by vadi2** [comment]
Yes exactly, the main window is not affected - the problem is about any secondary window.

I haven't found a Qt bug report about it yet.

**Comment by vadi2** [comment]
As far as I see, every Qt5 macOS app is affected by this. Filed https://bugreports.qt.io/browse/QTBUG-59433.

**Comment by bjorn** [comment]
Judging by the linked issue, this problem will be fixed once the Tiled builds are updated to use at least Qt 6.3.
